core/src/main/scala/com/codacy/analysis/core/tools/Tool.scala

```diff
@@ -158,47 +157,41 @@ class ToolCollector(toolRepository: ToolRepository) {
   }
 
   def fromNameOrUUID(toolInput: String, languages: Set[Language]): Either[AnalyserError, Set[Tool]] = {
-    from(toolInput, languages)
+    toolRepository.getTool(toolInput).flatMap(from(_, languages))
   }
 
   def fromToolUUIDs(toolUuids: Set[String], languages: Set[Language]): Either[AnalyserError, Set[Tool]] = {
     if (toolUuids.isEmpty) {
       Left(AnalyserError.NoActiveToolInConfiguration)
     } else {
-      val toolsIdentified = toolUuids.flatMap { toolUuid =>
-        from(toolUuid, languages).fold(
-          { _ =>
-            logger.warn(s"Failed to get tool for uuid:$toolUuid")
-            Set.empty[Tool]
-          },
-          identity)
+      toolRepository.listSupportedTools().map { tools =>
+        val toolsIdentified: Set[Tool] = tools.map { tool =>
+          if (toolUuids.contains(tool.uuid)) {
+            from(tool, languages) match {
+              case Left(_) =>
+                logger.warn(s"Failed to get tool for uuid:${tool.uuid}")
```

Comment:
Would it be interesting to have a message based on the type of the left? Or that kind of error will still show to the user? Because here, even if it is a Standalone, will just print a generic "failed to get tool"